src/main/resources/org/csanchez/jenkins/plugins/kubernetes/PodTemplate/help-yamlMergeStrategy.html

```diff
@@ -0,0 +1,6 @@
+Defines how the raw yaml field gets merged with yaml definitions from inherited pod templates.
+
+<ul>
+    <li>Override: The yaml definitions from inherited pod templates are completely overridden by the current yaml field.</li>
```

Comment:
Put this documentation in `help.html` for each option.

Reply:
Still recommend this ^^^

Reply:
Hum... does it work with `dropdownDescriptorSelector` ? The only place where I've seen this pattern being used was with `descriptorRadioList`. I could probably pull the description for each implementation in its own config page though.

Reply:
Not sure. Try `config.jelly` plus `f:description`. It is possible `f:dropdownDescriptorSelector` does not yet honor `help.html`, I do not recall.

Reply:
Yes, this is what I ended up doing in https://github.com/jenkinsci/kubernetes-plugin/pull/538/commits/e01f122916f09230b9e7ab5cc6d55ac770abac56